### PR TITLE
[Chore] Workflow to Alert Newcomers on Slack

### LINKED ISSUE #2403: [Chore] Create a gh workflow to notify about beginner-friendly issue on slack.
**Current Behavior**

We currently don't have an _automation_ that sends out a message in slack's `#newcomers` channel about beginner friendly issues when one (beginner-friendly issue) is created.

**Desired Situation**

A `gh` workflow needs to be created that posts a message in the `#newcomers` channel if any new issue is marked with `newcomers-only` or `good-first-issue`

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by theBeginner86**
**Description**
- This PR adds a workflow to notify about good first issues on the #newcomer channel of Layer5.
- This would get triggered only when a new or an already existing issue is labeled with `good first issue` or `newcomers-only` label
- This is reusable as it doesn't operate on the hardcoded repo name from which the issue was created.

This PR fixes #2403 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 3dfe9f679bebaccd7a8ae78691643966fcddbcc4 at: https://61b271180619ce12eb6a453a--layer5ng.netlify.app

File: .github/workflows/newcomers-alert.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    if: contains(github.event.issue.labels.*.name, 'good first issue') || contains(github.event.issue.labels.*.name, 'newcomers-only') 
+    name: Notify Slack for new good-first-issue
+    runs-on: ubuntu-latest
+    steps:
+      - name: Notify slack
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+        uses: pullreminders/slack-action@master
+        with:
+          args: '{\"channel\":\"C019426UBNY\",\"text\":\"A good first issue label was just added to ${{github.event.issue.html_url}}.\"}'
```
Comment (leecalcote) [inline .github/workflows/newcomers-alert.yml:16]: Fancy of you to grab the #newcomers channel id. Good work, @theBeginner86

File: .github/workflows/newcomers-alert.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+name: Newcomers Alert
+on:
+  issues:
+    types: [opened, labeled]
+jobs:
+  good-first-issue-notify:
+    if: contains(github.event.issue.labels.*.name, 'good first issue') || contains(github.event.issue.labels.*.name, 'newcomers-only')
```
Comment (leecalcote) [inline .github/workflows/newcomers-alert.yml:7]:
```suggestion
    if: contains(github.event.issue.labels.*.name, 'good first issue') || contains(github.event.issue.labels.*.name, 'first-timers-only') 
```

**Review by leecalcote — APPROVED** [review]
Way to take this on, @theBeginner86! 💯

File: .github/workflows/newcomers-alert.yml (hunk 1 above)
Comment (theBeginner86) [reply .github/workflows/newcomers-alert.yml:16]: I did a little bit of research to grab that id
Thanks 😁

**Comment by theBeginner86** [comment]
@leecalcote 
Thank you, for jumping in and fixing that label name

**Pull request merged**